app/assets/stylesheets/styles/pages.scss

```diff
@@ -306,11 +320,15 @@
 }
 
 .page-footer {
+  position: fixed;
```

Comment:
Properties should be ordered background-color, bottom, box-shadow, color, margin-bottom, position, width, z-index